#109 - public folder brought back

brzozka opened the pull request:
This PR should close #109 

As mentioned in Issue I didn't pay attention and accepted changes that caused problems in frontend side.

plencka commented:
Is the `public` folder even needed for some technical reason? We separated frontend and backend for a reason, and it seems weird that some front thing is lying around the main repo folder.

I'm not sure if current deploy works purely from cache or if site is build from templates.

I'm not frontend person by any means, but I understood that we either use dist or public, as both seem to serve same purpose. Both folder purpose seems to be mainly distribution.

brzozka commented:
@plencka We didn't need a public folder, but we needed it's content. I moved main html file into template, icons into assets folder and removed public afterwards.

plencka commented on frontend/templates/index.html:
```diff
@@ -0,0 +1,55 @@
+    </noscript>
+    <div id="app"></div>
+  </body>
+</html>
```
```suggestion
</html>

```

Please, add new line at the end of the file as per general used standard. Github highlights them with red circle with a dash.

You should be able to commit this suggestion directly to avoid losing time with pushes.
